packages/core/src/managed/environmentBaseElements/BaseEnvironment.less

```diff
@@ -38,6 +39,18 @@
       font-weight: 700;
     }
 
+    .env-deleting {
+      position: absolute;
+      right: 8px;
+      font-size: 13px;
+      color: var(--color-white);
+      animation: deletingAnimation 2s infinite;
+      padding: var(~'--3xs-spacing') var(~'--xs-spacing');
```

Comment:
Just for my education, what's the `~` here?

Reply:
LESS doesn't like the format `--3xs-spacing`, the `~` tells it to treat it as a string. It'll work without it but the UI will complain